Problem2.java

```diff
@@ -0,0 +1,48 @@
+//W - capacity, wt - weights array, v - values array, res - max value
+//Time - exponential
```

Comment:
Exponential Time Complexity is a good trick..For the PRS, do elaborate..You can always count the number of choices as the base and the recursion tree depth as the power.